sagemaker-fundamentals/create-training-job/create_training_job_vpc.ipynb

```diff
@@ -0,0 +1,869 @@
+{
```

Comment:
<p>Do the network interfaces get deleted? There is a comment but no code after it.</p><p>Missing closing paren on the last line. Should be <code>vpc_cleanup(vpc_res["Vpc"]['VpcId'])</code>.</p><p>Also style nit, you can be consistent about using either single or double quotes.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>the network interface is deleted under #delete subnets. but the comment is indeed confusing</p>